challenge.rb

```diff
@@ -1,26 +1,32 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  output = input.map {|x| x.capitalize}
+  return output
```

Comment:
You're right, I don't need the return statements here.  I just added it to make it easier to read and since it would be needed in some other languages too